.github/workflows/release-github.yml

```diff
@@ -31,9 +32,21 @@ jobs:
         with:
           go-version: '1.20.5'
       - name: run goreleaser
-        uses: goreleaser/goreleaser-action@v2
+        uses: goreleaser/goreleaser-action@v4
         with:
           version: latest
           args: release --rm-dist
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Extract version
+        id: extract-version
+        run: |
+          printf "::set-output name=%s::%s\n" tag-name "${GITHUB_REF#refs/tags/}"
+      - name: bump homebrew formula
+        uses: mislav/bump-homebrew-formula-action@v2
+        with: 
+          formula-name: oras
+          formula-path: Formula/oras.rb
+          download-url: https://github.com/oras-project/oras/archive/${{  steps.extract-version.outputs.tag-name  }}.tar.gz
```

Comment:
Instead of composing a download-url, you can specify `tag-name` directly. 

See: https://github.com/mislav/bump-homebrew-formula-action#action-inputs